src/views/student_view.py

```diff
@@ -25,7 +23,7 @@ def register_step2(self):
             if username:  # Ensure the input is not empty or just whitespace
                 return username
             else:
-                Color.prRed("\tName cannot be empty. Please enter a valid name.")
+                print("Name cannot be empty. Please enter a valid name.")
```

Comment:
Hey bro can you help me to return to  Color.prRed("\tName cannot be empty. Please enter a valid name.")